### DISCUSSION TITLE
关于流派的用法。我用emby听有声书，里边都有风格分类，还有音乐标签都是风格，没有流派。所以能否流派支持风格。

### DISCUSSION THREAD

**Discussion by ZH1007844330** [gitbobobo/StreamMusic#661] [Ideas]
或者在添加一个风格模块、

**Comment by ZH1007844330** [comment]
我找到了一个其他软件，和音流几乎差不多，他的流派就支持风格分类，而且有个比较好的点，就是可以隐藏不需要的项目，比如本地音乐，专辑艺术家。，或者文件夹，他的UI没音流做的好，我觉得，这2个功能，音流加上，就完美了。

**Reply by cndoin** [reply]
什么软件？？？？介绍一下

**Reply by ZH1007844330** [reply]
symfonium
好像只能谷歌下载最新版本

**Reply by cndoin** [reply]
没事我都能下载
告诉我名字嘛｡°(°¯᷄◠¯᷅°)°｡
